src/cdk/portal/dom-portal-outlet.ts

```diff
@@ -22,7 +22,8 @@ import {BasePortalOutlet, ComponentPortal, TemplatePortal} from './portal';
  */
 export class DomPortalOutlet extends BasePortalOutlet {
   constructor(
-      private _hostDomElement: Element,
+      /** Element into which the content is projected. */
+      public hostDomElement: Element,
```

Comment:
Maybe rename this to `outletElement`? I think the "host" is leftover from the old class name